tests/test_api_evolution_security.py

```
@@ -0,0 +1,138 @@
+)
+
+def run_async(coro):
+    return asyncio.run(coro)
+
+# Mock paths
+mock_runs_root = Path("/tmp/mock_runs")
+mock_repo_root = Path("/tmp/mock_repo")
+
+def test_get_run_evolution_patches_traversal():
+    """Test that path traversal in run_id is rejected."""
+    async def _test():
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `_test`

Add return type annotation: `None`

(ANN202)